ImageCopy.java

```diff
@@ -0,0 +1,23 @@
+package task7;
+
+import java.io.FileInputStream;
+import java.io.FileOutputStream;
+import java.io.IOException;
+
+public class ImageCopy {
+
+	public static void main(String[] args) throws IOException {
+		
+		System.out.println("hello");
+		
+        
+       FileInputStream in=new FileInputStream("/Users/sathish_s/Documents/images.jpeg");
```

Comment:
It's a good practice to use descriptive and meaningful variable names. can you please update them?